src/styles/index.css

```
@@ -0,0 +1,12 @@
+body {
+  margin: 0;
+  font-family: 'roboto', 'Montserrat'
```

Nit-picky, but could we fix the casing and add a fallback, e.g.:

```suggestion
  font-family: 'Roboto', 'Montserrat', sans-serif;
```